lib/crono_trigger/worker.rb

```diff
@@ -96,9 +103,9 @@ def heartbeat
       CronoTrigger::Models::Worker.connection_pool.with_connection do
         begin
           worker_record = CronoTrigger::Models::Worker.find_or_initialize_by(worker_id: @crono_trigger_worker_id)
-          worker_record.max_thread_size = @executor.max_length
-          worker_record.current_executing_size = @executor.scheduled_task_count
-          worker_record.current_queue_size = @execution_counter.value
+          worker_record.max_thread_size = CronoTrigger.config.executor_thread
+          worker_record.current_executing_size = @execution_counter.value
```

Comment:
I guess `current_executing_size` means the number of threads that are processing a record, and if so, `@execution_counter.value` is what you expect. `@executor.scheduled_task_count` is the total number of scheduled tasks since `@executor` is created.